File: app/src/main/java/org/imozerov/streetartview/ui/model/AuthorUi.java
package org.imozerov.streetartview.ui.model;

import org.imozerov.streetartview.storage.model.RealmAuthor;

/**
 * Created by imozerov on 05.02.16.
 */
public class AuthorUi {
    public final String id;
    public final String name;
    public final String description;

    public AuthorUi(String aId, String aName, String aDescription) {
        id = aId;
        name = aName;
        description = aDescription;
    }

    public AuthorUi(RealmAuthor realmAuthor) {
        this.id = realmAuthor.getId();
        this.name = realmAuthor.getName();
        this.description = realmAuthor.getDescription();
    }
}

File: app/src/main/java/org/imozerov/streetartview/ui/model/AuthorUi.kt
package org.imozerov.streetartview.ui.model

import org.imozerov.streetartview.storage.model.RealmAuthor

/**
 * Created by imozerov on 05.02.16.
 */
class AuthorUi {
    val id: String
    val name: String
    val photoUrl: String

    constructor(aId: String, aName: String, aPhoto: String) {
        id = aId
        name = aName
        photoUrl = aPhoto
    }

    constructor(realmAuthor: RealmAuthor) {
        id = realmAuthor.id
        name = realmAuthor.name
        photoUrl = realmAuthor.photo
    }
}
